website/content/en/docs/reference/configuration/unit-tests.md

```diff
@@ -0,0 +1,384 @@
+---
+title: Unit testing Vector configurations
+short: Unit tests
+weight: 5
+aliases: [
+  "/docs/reference/tests",
+  "/docs/reference/configuration/tests",
```

Comment:
I'm curious if we think this should still be under the configuration section, or if it's worth being on it's own

Reply:
As it's currently a solo document, I'd say best to keep it under configuration. That's easy to tweak later on tho.

Reply:
Just thought about it as didn't seem _directly_ related to config, but makes sense to move it if it gets larger